src/js/popup/view/timerCounterView.js

```diff
@@ -0,0 +1,20 @@
+import Timer from 'easytimer.js';
+
+const timer = new Timer();
+function updateTimerCounter(element, timer) {
+  element.innerText = timer.getTimeValues().toString();
```

Comment:
Do you think it's ok that we have startTime separately from this timer?
I mean what if we get 1 second difference between them? We will show 1 sec, but log 2 seconds to gitlab. I think it's possible.

Consider updating this new element with actual time spent. We can get current state here and calculate spent time.

Reply:
Good idea, store must be a single source of truth. Refactored.